docs/getting-started/installation.md

```diff
@@ -26,6 +26,12 @@ Prefect requires Python 3.8 or newer.
 
 We recommend installing Prefect using a Python virtual environment manager such as `pipenv`, `conda`, or `virtualenv`/`venv`.
 
+You can use [Prefect Cloud](/ui/cloud/) as your API server or [host your own Prefect server instance](/host/) backed by [PostgreSQL](/concepts/database/#configuring_a_postgresql_database).
+For development, you can use [SQLite](/concepts/database/#configuring_a_sqlite_database) 2.24 or newer as your database.
+
+[Prefect Cloud](/cloud/) is a managed solution that provides strong scaling, performance, and security.
+[Paid tiers](/cloud/users/) offer the user and team management features that enterprises require.
```

Comment:
```suggestion
```

Reply:
I don't think we need this info here.

Reply:
Trying to be responsive to comments to highlight differences between OS & Cloud. Can remove, but I think it might be nice to highlight for folks on a page that many folks find early in their journey.

Reply:
Maybe we can link out to another resource that explains the difference?

Reply:
Updated. What do you think about this treatment?

Reply:
Looks good